docs/qprofiler2/index.md

```diff
@@ -0,0 +1,328 @@
+                                only output HD and SQ.                                                  
+--bam-index                   Opt, a BAI index file. The option of "threads-producer" will be ignored   
+                                if no index file provided.                                              
+--bam-records <Integer>       Opt, only process the specified number of records from the beginning of   
+                                the BAM file. Default to process entire BAM file.                       
+--bam-validation              Opt, BAM record validation stringency [STRICT,LENIENT,SILENT].            
+                                Def=LENIENT.                                                            
+--help                        Show usage and help.                                                      
+--input                       Req, input file in FASTQ, BAM/SAM or VCF format, also support FA, MA,     
+                                QUAL, FASTA, GFF and GFF3 format.                                       
+--log                         Req, Log file.                                                            
+--loglevel                    Opt, Logging level [INFO,DEBUG], Def=INFO.                                
+--output                      Opt, XML output which containing basic summary statistics. Def="qprofiler.
+                                xml".                                                                   
+--threads-consumer <Integer>  Opt, number of threads to process inputed records, BAM and FASTQ mode     
+                                only. Def=0 (not multi-thread mode).                                    
+--threads-producer <Integer>  Opt, number of threads to read the indexed BAM file, BAM mode only. This  
+                                option will be ignored without "threads-consumer" option.  Def=1.       
+--vcf-format-field            Opt, group VCF records according to user specified format fields. eg.     
+                                "GD", "CCM","FT=PASS". Default to ignore VCF format field.              
+--version                     Show version number.     
+~~~~
+
+
+### multi thread
+When running multi-threaded, we suggest more consumer than producer threads with a recommended ratio of 6:1. However, it is up to your machine system. for example, only one thread to read the input file but 12 threads are specified to process reads.
+
+Please specify the BAM index file if multiple producer threads are going to be used, eg. 
+
+~~~~{.text}
+java -jar qprofiler2.jar --threads-consumer 12 -threads-producer 2 --bam-index  $somedir/${bam}.bai --input  $somedir/$bam --output $somedir/${bam}.qp2.xml --log $somedir/${bam}.qp2.log
+~~~~
+
+## Output
+
+### Xml Validation
+
+qprofiler2 provide a schema file which help you to validate the xml output. This xsd file can download from https://purl.org/adamajava/xsd/qprofiler2/v2/qprofiler2.xsd
```

Comment:
"qprofiler2 provides a schema file to help..."
"This xsd file can be download..."
url doesn't resolve

Reply:
why the url doesn't resolve?  I tried, it can access from outside or inside qimr

Reply:
It only resolves if you are vpn'ed in. Is this documentation purely for QIMRB people? If so, all good. If it is intended for external users, then it doesn't work.

Reply:
which url should we use for external user?

Reply:
sorry, don't know